### ISSUE TITLE
About the accuracy of Ghost-ResNet-50 (s=2) on Imagenet

### ISSUE THREAD

**Issue by JiaxiongQ** [iamhankai/ghostnet.pytorch#27]
How did you train Ghost-ResNet-50 (s=2) on Imagenet to achieve 75.0% top-1 accuracy?
Did you use cosine learning rate and label smooth?

**Comment by iamhankai** [comment]
We used multi-step learning rate without label smooth.
